Utilities/CMake/Linux/README.md

```diff
@@ -0,0 +1,4 @@
+# libHttpClient Linux CMake build system
```

Comment:
Before I look too deep into the CMake files (which I'm by no means an expert on)
Doesn't libhc already have a cmake file? Why aren't we using it?

Reply:
Not really. There are CMake files for android but for the other platforms it does not make cmake files but instead VCXProj files. the CMake.cmd script will go and copy the needed dependencies for each project and generate a list of files then it would paste them into a vcxproj template.

Reply:
We still have a CMakeList.txt file in Utilities/CMake/ is there a way we can use it as a source for the linux build?
The current setup is already painful enough to use without adding another project file nobody will remember to ever update or test.

Reply:
@lubeltra I'm fairly certain that CMake exists for VS CMake integration, looking over it, integrating Linux into it may be more trouble than it's worth. I do get your point though - I do think there might be value in combining the Linux and Android CMakes into an integrated one. Might be out of scope for this PR though.